app/webhook_test.go

```diff
@@ -308,21 +300,14 @@ func TestCreateWebhookPost(t *testing.T) {
 		},
 		"webhook_display_name": hook.DisplayName,
 	}, model.POST_SLACK_ATTACHMENT, "")
-	if err != nil {
-		t.Fatal(err.Error())
-	}
+	require.Nil(t, err)
 
 	for _, k := range []string{"from_webhook", "attachments", "webhook_display_name"} {
```

Comment:
Probably here makes sense to just use something like 
```suggestion
	assert.Contains(t, post.Props, "from_webhook", "missing from_webhook prop")
	assert.Contains(t, post.Props, "attachments", "missiong attachments prop")
	assert.Contains(t, post.Props, "webhook_display_name", "missing webhook_display_name prop")
```